README.md

```diff
@@ -1,5 +1,7 @@
 # PicturePipe Renovate Preset Config
 
+[![CircleCI Build](https://circleci.com/gh/PicturePipe/renovate-config.svg?style=shield&circle-token=007d01de72a5dbb45326b110030928d57ffb0220)](https://circleci.com/gh/PicturePipe/renovate-config "CircleCI Build")
```

Comment:
This is a public repository, so we don't need the `token`.

Reply:
Good catch. I have fixed it and removed the API token from CircleCI.